app/assets/stylesheets/comfortable_mexican_sofa/admin/layout/_content.scss

```diff
@@ -0,0 +1,19 @@
+// Layout - Content
+//
+// Styleguide Layout - Content
+
+.l-split-content {
+  @include row(12);
+}
+
+.l-split-content__col {
+  & + & {
```

Comment:
In Sass, `&` refers to the current selector, so `& + &` would output as `.l-split-content__col + .l-split-content__col`.